tests/tests/datm_ocn_ice_mx025_gefs

```diff
@@ -0,0 +1,63 @@
+#
+#  CPLD_DATM_MOM6_CICE_CMEPS_1_STEP_COLD test
```

Comment:
Please add _GEFS here to indicate this uses GEFS forcings.

Reply:
Added _GEFS.